packages/worker/src/metadata.ts

```diff
@@ -0,0 +1,27 @@
+export function readMetadata(buffer: Buffer, offset: number) {
```

Comment:
add a file-level comment explaining what this metadata is

Reply:
I have already changed the file name to `size` and added the documentation to clear up confusion and not invent a slang word in the bazel world.